selfdrive/controls/lib/latcontrol_torque.py

```diff
@@ -27,7 +27,7 @@ def __init__(self, CP, CI):
     self.torque_params = CP.lateralTuning.torque
     self.pid = PIDController(self.torque_params.kp, self.torque_params.ki,
                              k_f=self.torque_params.kf, pos_limit=self.steer_max, neg_limit=-self.steer_max)
-    self.torque_from_lateral_accel = CI.torque_from_lateral_accel()
+    self.torque_from_lateral_accel = CI.torque_from_lateral_accel(self.torque_params.linear)
```

Comment:
@MoreTore wondering if it's possible to use stock controller or create a new one like latcontrol_ti?
comma team still progressively updates it so it may be different in each version, doing so we can:
1. not affecting other vehicles
2. Consistent experiences for all TI users.

Reply:
This would not affect other vehicles. The default is false